pkg/pillar/containerd/containerd.go

```diff
@@ -914,3 +951,28 @@ func bindNS(ns string, path string, pid int) error {
 	}
 	return nil
 }
+
+func newServiceCtx(namespace string) (context.Context, context.CancelFunc) {
+	return context.WithCancel(namespaces.WithNamespace(context.Background(), namespace))
+}
+
+func newServiceCtxWithLease(ctrdClient *containerd.Client, namespace string) (context.Context, context.CancelFunc, error) {
+	if ctrdClient == nil {
+		return nil, nil, fmt.Errorf("newServiceCtxWithLease(%s): exception while verifying ctrd client: "+
+			namespace, "Container client is nil")
+	}
+	ctx, cancel := newServiceCtx(namespace)
+	ctx, done, err := ctrdClient.WithLease(ctx)
```

Comment:
Can't we inline this code so we end up with a single cancel/done function instead of two such functions?

Reply:
What @eriknordmark said ;-)

Reply:
Added a comment as discussed.